README.md

```diff
@@ -37,7 +37,9 @@ A searchable sort-able and group-able version of the list of companies below is
 - [IRESS](https://www.iress.com/au/company/join-us/) | Melbourne CBD, Melbourne; Sydney CBD, Sydney | Reactjs, Python, JavaScript, TypeScript, AWS
 
 ## J - L
-- [Koala](https://www.linkedin.com/company/koala-mattress/jobs/) | Alexandria, Sydney | Go, PHP, Javascript, AWS
+- [JXT Global](https://stackoverflow.com/jobs/companies/jxt-global) | Sydney CBD, Sydney | .NET, C#, React, Angular, AWS
+- [Kayo Sports](https://stackoverflow.com/jobs/companies/kayo-sports) | Artamon, Sydney | HTML, Javascript, Css, Node.Js, React.Js, Express.js 
+- [Koala](https://www.linkedin.cohttps://stackoverflow.com/jobs/companies/kayo-sportsm/company/koala-mattress/jobs/) | Alexandria, Sydney | Go, PHP, Javascript, AWS
```

Comment:
I guess you have messed this up by mistake.

Reply:
oh shit. Yep. Will fix it